Dockerfile

```diff
@@ -61,11 +84,21 @@ RUN chown -R $APP_USER:$APP_USER $APP_DATA
 COPY packaging/conf/database.yml ./config/database.yml
 
 # Run the npm postinstall manually after it was copied
-RUN DATABASE_URL=sqlite3:///tmp/db.sqlite3 RAILS_ENV=production bundle exec rake assets:precompile
+RUN DATABASE_URL='nulldb://nohost' RAILS_ENV=production bundle exec rake assets:precompile
 
 # Include pandoc
-RUN DATABASE_URL=sqlite3:///tmp/db.sqlite3 RAILS_ENV=production bundle exec rails runner "puts ::OpenProject::TextFormatting::Formats::Markdown::PandocDownloader.check_or_download!"
+RUN DATABASE_URL='nulldb://nohost' RAILS_ENV=production bundle exec rails runner "puts ::OpenProject::TextFormatting::Formats::Markdown::PandocDownloader.check_or_download!"
+
+# Remove node_modules and entire frontend
+RUN rm -rf $APP_PATH/node_modules/ $APP_PATH/frontend/node_modules/
+
+# Re-own app dir
+RUN chown -R $APP_USER:$APP_USER $APP_PATH
```

Comment:
This is also an expensive layer. We might want to switch to setting up everything as root and switching to the APP_USER on startup using gosu.

Reply:
I'm fine with any change as long it ensures that each service is being run with the correct permissions, which did not take place in the 8.3. build